recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - openff-interchange >=0.4  # if we need 0.3, can relax later
     - openff-toolkit >=0.16.1
     - openff-units >=0.2
-    - openmm >=7.6.0
+    - openmm >=8.1.2
```

Comment:
This is because of https://github.com/conda-forge/openmmforcefields-feedstock/issues/23